internal/rpc/client/client.go

```diff
@@ -48,7 +48,7 @@ func (rc *RPCClient) Start() error {
 		opts = append(opts, grpc.WithTransportCredentials(tlsCreds))
 	}
 
-	cc, err := grpc.Dial(fmt.Sprintf("%s:%d", rc.cfg.Host, rc.cfg.Port), opts...)
+	cc, err := grpc.NewClient(fmt.Sprintf("%s:%d", rc.cfg.Host, rc.cfg.Port), opts...)
```

Comment:
Why this change? New lib version?

Reply:
Yes,
the upgrade form v1.62.2 -> v1.64.1
 Dail is deprecated
https://pkg.go.dev/google.golang.org/grpc@v1.64.1#Dial
https://pkg.go.dev/google.golang.org/grpc@v1.64.1#NewClient

Reply:
I tried to avoid using `go mod tidy or `go get -u` so I'm not sure how this package got upgraded - perhaps as one of the dependencies?

Reply:
It looks fine to me If the new versions doesn't break anything.